src/theme.ts

```diff
@@ -95,7 +95,8 @@ const colors = {
 const defaultControlHeight = 36;
 const emphasizedControlHeight = 48;
 
-export const breakpoints = ['32em', '48em', '64em', '80em'];
+export const breakpoints = ['576px', '768px', '992px', '1200px'];
+export const containers = ['540px', '720px', '960px', '1140px'];
```

Comment:
Why do we have a separate list of `containers` breakpoints here? it seems completely redundant - can we not just make them the same?

Reply:
But that's what we were doing before. We would abstract an arbitrary number from the breakpoint to calculate the width.

That's what I'm going for:

![Screen Shot 2019-07-12 at 4 20 20 PM](https://user-images.githubusercontent.com/4951004/61131105-f9045e80-a4c0-11e9-855c-d1bb298a5433.png)

Reply:
Make the breakpoints numbers, and then apply a conversion to reduce the number in the container component.

Reply:
Styled system expects a unit, so you can just use `calc` as before.

Reply:
Why not give the option to the user the change the container size?

Reply:
The container size is meant to be coupled with the breakpoints, plus some margin, right? I think in order to answer this we need to know what exactly the container is meant to do.

Reply:
I took these breakpoints/container sizes from bootstrap as a reference. 

Considering there isn't a clear formula for the max-width calculation, I decided to include them in a `container` array.

My use-case is to emulate the previous behavior of the container, without having the side effects we do now.

If we want to use a specific magic number for the calculation, I don't have any objection. Current we extract the values from the `theme.space` array which makes no sense.